scripts/jimmy

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
more generic way. "/usr/bin/python" was pointing to the wrong version of python for me.